src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/PensionAnnexDateStampService.java

```diff
@@ -0,0 +1,85 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.pdfbox.Loader;
+import org.apache.pdfbox.pdmodel.PDDocument;
+import org.apache.pdfbox.pdmodel.interactive.form.PDAcroForm;
+import org.apache.pdfbox.pdmodel.interactive.form.PDField;
+import org.apache.pdfbox.pdmodel.interactive.form.PDTextField;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+import uk.gov.hmcts.reform.finrem.caseorchestration.error.DocumentStorageException;
+import uk.gov.hmcts.reform.finrem.caseorchestration.error.StampDocumentException;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.FinremMultipartFile;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.document.Document;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.evidence.FileUploadResponse;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.evidencemanagement.EvidenceManagementDownloadService;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.evidencemanagement.EvidenceManagementUploadService;
+
+import java.io.ByteArrayOutputStream;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Collections;
+import java.util.List;
+import java.util.Locale;
+import java.util.Optional;
+
+import static java.lang.String.format;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.service.DocumentManagementService.CONVERTER;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class PensionAnnexDateStampService {
+
+    public static final String APPLICATION_PDF_CONTENT_TYPE = "application/pdf";
+    private final EvidenceManagementUploadService emUploadService;
+
+    private final EvidenceManagementDownloadService emDownloadService;
+
+    public static final String FORM_P1_DATE_OF_ORDER_TEXTBOX_NAME = "Date the court made/varied/discharged an order";
```

Comment:
You don't need public here. Just use 
`static final String FORM_P1_DATE_OF_ORDER_TEXTBOX_NAME = "Date the court made/varied/discharged an order";`